killuazhu/worker
# Add option to supply extra hosts

## What is the problem that this PR is trying to fix?

Add a custom DNS entry in worker

## What approach did you choose and why?

Extra option in worker binary. 

Alternatives considered
- Apply globally at docker daemon, did not find a way to do so
- Custom DNS server, seems overkill

## How can you test this?

Launch the worker locally using file based queue, verify docker extra host options are added to the running container through `docker inspect`

## What feedback would you like, if any?
